main.go

```diff
@@ -61,7 +72,7 @@ func initialize(unitTestfunc callback, configOverride *config.Config) {
 	guestProcess, err := pty.CreateGuestProcess(shellStr)
 	if err != nil {
 		pty.Close()
-		logger.Fatalf("Failed to start your shell: %s", err)
+		logger.Fatalf("Failed to start your shell \"%s\": %s", shellStr, err)
```

Comment:
you could just use `%q` here instead of inserting the quotes manually

Reply:
fixed in https://github.com/liamg/aminal/pull/262/commits/fa710446a787725f71301a1f8b2e2450dc4c42ab